Makefile

```diff
@@ -8,106 +8,99 @@ ARGS:=
 TIMEOUT:=90
 
 
-DOCKER=docker-compose \
-	--env-file .env \
+CONTAINER_APP=podman-compose \
+	--env-file=.env \
```

Comment:
The weird thing is that the argument doesn't appear in the *podman-compose --help*, but it is in the [documentation](https://docs.podman.io/en/latest/markdown/options/env-file.html).

Reply:
Yep, that is why I am creating the podman-compose nightly pkg